blog/docs/articles/knative-eventing-vision.md

```diff
@@ -0,0 +1,88 @@
+
+**Date: 2023-07-27**
+
+## What is next in Knative Eventing?
+
+Knative Eventing has come a long way and has established itself as the goto platform on Kubernetes for event-driven applications. However we are not done with the project and we continue to evolve it. We are improving existing APIs and adding new features to Knative Eventing it as well. Below is a high level overview of what we are currently doing for the next couple of months.
+
+Please keep in mind that we need your feedback as well! If there are missing features or needed improvements, please speak up and tell.
+
+## Secure Event Processing
+
+Secure event processing refers to the practice of processing and analyzing events or data streams in a secure and trusted manner. Events can include various types of data, such as user interactions, system logs, sensor readings, network traffic, or any other form of real-time or near-real-time data.
+
+The primary goal of secure event processing is to ensure the confidentiality, integrity, and availability of the processed events while maintaining the privacy and security of the underlying data.
+
+### Transport encryption
+
+Currently, event delivery within the cluster is unencrypted. This limits the types of events which can be transmitted to those of low compliance value (or a relaxed compliance posture) or, alternatively, forces administrators to use a service mesh or encrypted CNI to encrypt the traffic, which poses many challenges to Knative Eventing adopters.
+
+Knative Brokers and Channels will provide HTTPS endpoints to receive events. Given that these endpoints typically do not have public DNS names (e.g. svc.cluster.local or the like), these need to be signed by a non-public CA (cluster or organization specific CA).
+
+Event producers will be able to determine the CA root used to sign the Broker or Channel certificate from the Knative resource’s metadata.
+
+For more information, please refer to the [feature track document](https://docs.google.com/document/d/1H-x_oji8LqkCyd7tlsSyclmUe7FAmEJPgRxOU_0pkn8/edit?resourcekey%253D0-lzDIPJsZOP3G17QE_g1lHw#heading%253Dh.8eg1on3fge8w).
+
+### Sender identity
+
+Currently, event delivery within the cluster is unauthenticated, and an event consumer cannot determine the identity of any sender.
+
+By sending the identity of the event sender along with the events, event receivers can start implementing access control features.
+
+For more information, please refer to the [feature track document](https://docs.google.com/document/d/1e7UgNTkL0Br5Da09Rg2ieVmhKJo4VXuBj-mHT9NCujY/edit).
+
+### Authorization and admission policies
+
+Event-driven architecture is meant to lower organizations' silos and barriers, make systems more resilient and ultimately increase business agility, however, this is not possible without event admission policies and safeguards for security and data quality.
```

Comment:
All the other sections have a section about what the current state is. Wanna write something similar?

Even if that we don't have anything, and brokers+channels+etc will simply accept any events.

Reply:
@pierDipi can you take a look here?

I think it is a good suggestion by Ali